### DISCUSSION TITLE
matchUtilities is not a function in Just-in-Time Mode

### DISCUSSION THREAD

**Discussion by Demetryi** [tailwindlabs/tailwindcss#4051] [Help]
When I start using mode: "jit" I get 
```javascript
matchUtilities is not a function 
```
error, removing jit no error

**Comment by spencercal** [comment]
I'm experiencing the same issue. Having to run without jit enabled for now.

**Comment by Demetryi** [comment]
in my case it has to do with using xwind, hopefully they will update their package ...

**Comment by Firepanel** [comment]
I also noticed that when updating tailwind.config.js, it fire up hmr and you can see on console the updates to several components on page and also the index.css file, but you can't see the changes applied. Only if I save several times, this error comes up and then I can see the changes.
